src/repl.rs

```diff
@@ -308,6 +310,26 @@ impl ReplState {
                                 //     // FIXME: Never panic. Instead return Continuation::Error when evaluating.
                                 //     ()
                                 // }
+                            } else if s == &":ASSERT-EMITTED" {
+                                let (first, rest) = store.car_cdr(&rest);
+                                let (second, rest) = store.car_cdr(&rest);
+
+                                assert!(rest.is_nil());
+                                let (_, _, _, emitted) = self.clone().eval_expr(first, store);
+                                let (second_evaled, _, _, _) = self.eval_expr(second, store);
```

Comment:
By convention, I think the order should be reversed here:

`!(:assert-emitted '(1 2 3) (begin (emit 1) (emit 2) (emit 3)))`

That is, the expected result should come first.